tox.ini

```diff
@@ -54,3 +55,24 @@ commands: {[common-lint]commands}
 [testenv:py{37,38,39,310,311}-lint]
 extras: {[common-lint]extras}
 commands: {[common-lint]commands}
+
+[testenv:py{37,38,39,310,311}-wheel]
+basepython=
+    py37: python3.7
+    py38: python3.8
+    py39: python3.9
+    py310: python3.10
+    py311: python3.11
+deps=
+    wheel
+    build
+allowlist_externals=
+    /bin/rm
+    /bin/bash
+commands=
+    pip install --upgrade pip
+    /bin/rm -rf build dist
+    python setup.py sdist bdist_wheel
+    /bin/bash -c 'pip install --upgrade "$(ls dist/<MODULE_NAME>-*-py3-none-any.whl)" --progress-bar off'
+    python -c "from <MODULE_NAME> import encode"
```

Comment:
`encode` is not a general thing. This was likely copied from `eth-abi`.

This should maybe be `import <MODULE_NAME>`?

Reply:
Yup, should'a run it on a live repo before just pasting here :(

It's cleaned up and working on `py-ssz`, and the template now matches that.